### DISCUSSION TITLE
需要该书的源码

### DISCUSSION THREAD

**Discussion by kingwolfwuqun** [SpringFarSoft/Book#15] [Show and tell]
需要该书的源码

**Comment by troubleshooter001** [comment]
书中的QQ加不上，源码不知道哪里获取

**Comment by anjoy8** [comment]
在书的背面有获取资料的方法，资料有视频和源码，通过机械工业出版社的公众号二维码。

也可以加QQ群444011511

**Comment by boogaloo-jw** [comment]
> 需要该书的源码

https://github.com/SpringFarSoft/SwiftCode.BBS
